.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+charset = utf-8
+insert_final_newline = true
+trim_trailing_whitespace = true
+end_of_line = lf
+
+[*.txt]
+trim_trailing_whitespace = false
```

Comment:
> `insert_final_newline = true` 

The irony...

Reply:
I did not edit that file in VS to be fair 😆